src/config/Options.js

```diff
@@ -2991,6 +2991,7 @@ export default class Options {
 			 * - step-before
 			 * - step-after
 			 * @property {Boolean|Array} [line.point=true] Set to false to not draw points on linecharts. Or pass an array of line ids to draw points for.
+			 * @property {Boolean} [line.zerobased=true] Set if min or max value will be 0 on line chart.
```

Comment:
It should be `[line.zerobased=false]`, because zerobased's default value is false.